statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1559,6 +1559,52 @@ def test_binary_pred_table_zeros():
     assert_equal(res.pred_table(), expected)
 
 
+class TestGeneralizedPoisson(object):
+    """
+    Test Generalized Poisson model
+    """
+    @classmethod
+    def setupClass(cls):
+        data = sm.datasets.randhie.load()
+        data.exog = sm.add_constant(data.exog)
+        cls.res1 = GeneralizedPoisson(data.endog, data.exog, p=2).fit(method='newton')
+        res2 = RandHIE()
+        res2.generalizedpoisson_gp2()
+        cls.res2 = res2
+
+    def test_bse(self):
+        assert_almost_equal(self.res1.bse, self.res2.bse, DECIMAL_4)
+
+    def test_params(self):
+        assert_almost_equal(self.res1.params, self.res2.params, DECIMAL_4)
```

Comment:
It's better to use now `assert_allclose` which allows using absolute and relative tolerance instead of decimals, which is more flexible and precise if the magnitude of the test values varies a lot.
(The `assert_almost_equal` test are from before `numpy.testing.assert_allclose` was available.)